tutorials/01_get_started.ipynb

```diff
@@ -8,14 +8,18 @@
     "\n",
     "## Overview\n",
     "\n",
-    "🚀 In this tutorial, we'll initialize XRFeitoria and add basic shapes."
+    "🚀 This tutorial demonstrates a minimal application of XRFeitoria. By the end of this tutorial, you will be able to:\n",
+    "- Open Blender/Unreal Engine by XRFeitoria\n",
+    "- Import a mesh\n",
+    "- Add a camera\n",
+    "- Render the mesh and save the image"
```

Comment:
I prefer to stress the essential steps that are needed to complete this MVP.

```suggestion
    "🚀 This tutorial demonstrates a minimal application of XRFeitoria. To accomplish this, essential steps should be taken:
    - initialization
    - import an `actor`
    - add a `camera`
    - add a `sequence`
    - render
    
    [`actor`](https://xrfeitoria.readthedocs.io/en/latest/apis/actor.html)
, [`camera`](https://xrfeitoria.readthedocs.io/en/latest/apis/camera.html), and [`sequence`](https://xrfeitoria.readthedocs.io/en/latest/apis/sequence.html) are classes defined in `XRFeitoria`.  We'll introduce them in detail step by step.
```

Reply:
fixed in ce589b6